app/controllers/archive_faqs_controller.rb

```diff
@@ -104,7 +104,7 @@ def update
   # reorder FAQs
   def update_positions
     if params[:archive_faqs]
-      @archive_faqs = ArchiveFaq.reorder(params[:archive_faqs])
+      @archive_faqs = ArchiveFaq.reorder(params[:archive_faqs].map { |s| s.to_i.to_s })
```

Comment:
I'm fairly certain that this is a false positive. It looks like Brakeman thinks that this is calling the built-in Rails `reorder` function, which can be vulnerable to SQL injection if you pass arbitrary user input. But the `ArchiveFaq` model overrides the definition of `reorder` to do something completely different:
https://github.com/otwcode/otwarchive/blob/3862fc69353d3c9c6e06550ac860b6aa067170e5/app/models/archive_faq.rb#L30-L32
And I don't believe that definition of `reorder` is vulnerable to SQL injection, since it calls `collect!(&:to_i)` on its input:
https://github.com/otwcode/otwarchive/blob/3862fc69353d3c9c6e06550ac860b6aa067170e5/lib/sortable_list.rb#L2-L13
It seems to me that it might be cleaner to just rename the `reorder` function to `reorder_questions` to avoid confusion (both for Brakeman and for anyone viewing the code), instead of calling `to_i.to_s` everywhere.

Reply:
I am not disagreeing that it is probably a false positive I am trying to minimise the  number of issues brakeman raises so that we don't just list all the things as false positives and we are less likely to make a mistake if we do that.  So I would also be happy with "rename the reorder function to reorder_questions". I have been trying to make the smallest changes I could and I had the sortable list code to be imported from elsewhere.